implementations/rust/ockam/ockam_command/src/util/api.rs

```diff
@@ -157,7 +157,7 @@ pub struct TrustOpts {
 
     /// Expect credential manually saved to the storage
     #[arg(long)]
-    pub expect_cached_credential: bool,
+    pub credential_scope: Option<String>,
```

Comment:
What's the difference between the free-form scopes and the ones from `CredentialScope`?

Reply:
In the database they're strings and can have any value as long as it doesn't conflict with another credential. For example in bats tests I use "test" values. However, for project admin and project membership credentials I use a predefined format so we can identify where they belong. To be honest I don't think it's a perfect design, but we don't yet have a good use case for it either (except the bats tests)